Cosmetics.py

```diff
@@ -779,6 +779,11 @@ def patch_instrument(rom, settings, log, symbols):
     log.sfx['Ocarina'] = ocarina_options[choice]
 
 def patch_voices(rom, settings, log, symbols):
+    if settings.disable_custom_music:
```

Comment:
What's the logic behind this exactly? I feel there's a better condition to check for outputting a ZPF than this one that I guess is enabled at the same time (Also would this break voices on web since custom music is always disabled there?)

Reply:
`settings.disable_custom_music` is the "hey, we don't want to do this when outputting a ZPF" pseudo-setting. It could be renamed to be more clear but this is a proper usage of it, IMO.